README.md

```diff
@@ -1,70 +1,222 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Installation
+To get started with Math-Magician, follow these steps:
+
+```sh
+Clone the repository to your local machine.
+Navigate to the project directory in your terminal.
+Run npm install to install the project dependencies.
+Run npm start to start the development server.
+Open http://localhost:3000 to view the app in your browser.
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Run the app by running `npm start` in the terminal
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+    npm test
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+Example:
+
+```sh
+github pages
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shalini Nayan**
+
+- GitHub: [@ShaliniNayan](https://github.com/ShaliniNayan)
+- Twitter: [@sndprs](https://twitter.com/sndprs)
+- LinkedIn: [shalini-nayan-5820389](https://linkedin.com/in/shalini-nayan-5820389)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Future enhancements to this app could include:
+- Integration with other APIs or databases
+- Improved styling and user interface design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions to this project are welcome. To contribute, please fork this repository and submit a pull request with your changes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, follow and contact me back.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse community and my coding partner for their dedicated time to encourage me achieve the best. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Question_1: Have you used linters to check for bugs?**
+
+  - yes
+
+- **Question_2: Do the project use database?**
+
+  - Answer_2: not at the moment. To be used as we scale the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is licensed under the [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken, consider fixing it to make your readme professional.

Reply:
Thank you & Noted.